linux/tools.Dockerfile

```diff
@@ -37,6 +37,19 @@ RUN bash ./tdnfinstall.sh gh
 RUN mkdir -p /usr/cloudshell
 WORKDIR /usr/cloudshell
 
+# Install Office 365 CLI templates
+RUN npm install -q -g @pnp/cli-microsoft365
+
+# Install Bicep CLI
+RUN curl -Lo bicep https://github.com/Azure/bicep/releases/latest/download/bicep-linux-x64 \
+  && chmod +x ./bicep \
+  && mv ./bicep /usr/local/bin/bicep \
+  && bicep --help
+
+# Temp: fix ansible modules. Proper fix is to update base layer to use regular python for Ansible.
+RUN wget -nv -q https://raw.githubusercontent.com/ansible-collections/azure/dev/requirements-azure.txt \
+    && /opt/ansible/bin/python -m pip install -r /usr/share/ansible/collections/ansible_collections/azure/azcollection/requirements-azure.txt 
+    
```

Comment:
Since we did couple of base image releases, is the temp fix completed? Can we move this to base as well.